modules/client-configuration/pages/snippets/trust_gpg.adoc

```diff
@@ -74,7 +74,6 @@ You do not need to delete any previously stored keys.
 [IMPORTANT]
 ====
 Trusting a GPG key is important for security on clients.
-It is the task of the admin to decide which keys are needed and can be trusted.
-A software channel cannot be used when a GPG key is not trusted.
-Assigning a channel to a client depends on if its GPG key has been trusted.
+It is the task of the administrator to decide which keys are needed and can be trusted.
+Because a software channel cannot be used when the GPG key is not trusted, the decision of assigning a channel to a client depends on the decision of trusting the key.
```

Comment:
I think we could simplify the wording:
```suggestion
A software channel cannot be assigned to a client when the GPG key is not trusted.
```
Yesterday, I unfortunately wasn't inspired enough...  Maybe, we should also drop the negations here.